### ISSUE TITLE
Multi Project builds do not generate reports correctly

### ISSUE THREAD

**Issue by vx-bill** [serenity-bdd/serenity-gradle-plugin#13]
### **Problem**
When running the aggregate task from a gradle subproject the reports are not generated properly.  e.g missing feature names and broken links.
Environment: 
- `net.serenity-bdd.serenity-gradle-plugin:3.4.2 `
- `Gradle 7.5.1`
- Windows

### **Workaround**
Set the user.dir system property to the project directory before the aggregate task runs.  The aggregate task then creates the reports as expected.

Add the following to the subproject running serenity tasks:
```
tasks.withType<net.serenitybdd.plugins.gradle.AggregateTask> {
    doFirst { System.setProperty("user.dir", project.projectDir.toString()) }
}

```
### **Possible Problem location**
```
class SerenityAbstractTask extends DefaultTask {
    static Path absolutePathOf(Path path) {
        return Paths.get(System.getProperty("user.dir")).resolve(path)
    }
```
Should this method use the `user.dir` system property rather than the project directory?

**Comment by patrickvkb** [comment]
This seems to be resolved in the version 3.9.8

**Comment by zeners** [comment]
please close the issue, if resolved
